dashboard/scripts/update_tts_i18n_static_messages.rb

```diff
@@ -88,6 +111,7 @@
 
       content_hash = Digest::MD5.hexdigest(text)
       filename = "#{loc_voice[:VOICE]}/#{loc_voice[:SPEED]}/#{loc_voice[:SHAPE]}/#{content_hash}/#{text}.mp3"
+      text = TextToSpeech.sanitize(text)
```

Comment:
we should add a comment explaining why it's necessary to do this _after_ generating the content hash